<?xml version="1.0"?>
<robot name="decor"> <!-- puzzle environment -->
  <material name="Blue">
    <color rgba="0 0 1 1"/>
  </material>
  
  <link name="decor_base">
    <visual>
      <origin xyz="-0.575 0 -0.05"/>
      <geometry>
        <box size="0.85 2 0.1" />
      </geometry>
      <material name="Blue"/>
    </visual>
    <collision>
      <origin xyz="-0.575 0 -0.05"/>
      <geometry>
        <box size="0.85 2 0.1" />
      </geometry>
    </collision>
  </link>
  
  <joint name="j_decor_one" type="fixed">
    <parent link="decor_base"/>
    <child link="l_decor_one"/>
    <origin rpy="0 0 0" xyz="0 0 0"/>
  </joint>
        
  <link name="l_decor_one">
    <visual>
      <origin xyz="0.575 0 -0.05"/>
      <geometry>
        <box size="0.85 2 0.1" />
      </geometry>
      <material name="Blue"/>
    </visual>
    <collision>
      <origin xyz="0.575 0 -0.05"/>
      <geometry>
        <box size="0.85 2 0.1" />
      </geometry>
    </collision>
  </link>
  
  
  <joint name="j_decor_two" type="fixed">
    <parent link="l_decor_one"/>
    <child link="l_decor_two"/>
    <origin rpy="0 0 0" xyz="0 0 0"/>
  </joint>
        
  <link name="l_decor_two">
    <visual>
      <origin xyz="0 -0.7 -0.05"/>
      <geometry>
        <box size="0.3 0.6 0.1" />
      </geometry>
    <material name="Blue"/>
    </visual>
    <collision>
      <origin xyz="0 -0.7 -0.05"/>
      <geometry>
        <box size="0.3 0.6 0.1" />
      </geometry>
    </collision>
  </link>
  
  
  <joint name="j_decor_three" type="fixed">
    <parent link="l_decor_two"/>
    <child link="l_decor_three"/>
    <origin rpy="0 0 0" xyz="0 0 0"/>
  </joint>
        
  <link name="l_decor_three">
    <visual>
      <origin xyz="0 0.7 -0.05"/>
      <geometry>
        <box size="0.3 0.6 0.1" />
      </geometry>
      <material name="Blue"/>
    </visual>
    <collision>
      <origin xyz="0 0.7 -0.05"/>
      <geometry>
        <box size="0.3 0.6 0.1" />
      </geometry>
    </collision>
  </link>
</robot>
